### ISSUE TITLE
Programs in "bin" folder are too vague and do not include adequate documentation.

### ISSUE THREAD

**Issue by userse31** [LukeZGD/Legacy-iOS-Kit#330]
The included programs included in the "bin" folder in the root of the project oftentimes don't have terribly useful "--help" outputs.

Example: running "./bin/linux/x86_64/ipsw --help" returns with a fairly useless help section.

The listed switches (like '-e "<action to include>"') are not elaborated on, and the other switches are listed as "[-daibutsu]" which, needless to say, is NOT helpful.

Basically, I want manuals for the included command line utilities, _especially_ those that are extremely vague.

**Comment by LukeZGD** [comment]
For ipsw, refer to the daibutsuCFW readme: https://github.com/dora2-iOS/daibutsuCFW#how-to-make-custom-firmware

For more details, read the code: `ipsw-patch/main.c`

Or the script portion where ipsw is used in `restore.sh`

Same goes to the other tools: read their source code to know more details about their usage

I plan to make a documentation of the things that the script does someday, but it will take a lot of time

**Comment by userse31** [comment]
Documentation is planned? Nice!
